kyuubi-main/src/main/scala/org/apache/kyuubi/operation/KyuubiOperationManager.scala

```diff
@@ -49,6 +50,17 @@ class KyuubiOperationManager private (name: String) extends OperationManager(nam
     tSessionHandle
   }
 
+  private def getQueryTimeout(statementTimeout: Long, session: Session): Long = {
+    // If a timeout value `queryTimeout` is specified by users and it is smaller than
+    // a session timeout value, we use the user-specified value.
+    val sessionTimeout = getConf.get(KyuubiConf.OPERATION_QUERY_TIMEOUT)
```

Comment:
nit: rename session timeout to `systemQueryTimeout` and `statementTimeout` to `clientQueryTimeout`